pages/index.js

```diff
@@ -10,10 +18,33 @@ const WorkspaceContainer = dynamic(
   }
 )
 
-const Home = () => (
-  <Layout>
-    <WorkspaceContainer />
-  </Layout>
-)
+const Home = () => {
+  const router = useRouter()
```

Comment:
Should we move this code to _app.js so this works for all pages?

Reply:
I couldn't move it  there because you can only use AuthContext within the context of AuthContextProvider or it is undefined.  Might work with some significant refactoring.

Reply:
@mannycolon

Reply:
@PhotoNomad0 It could also be moved into the Payout component since all pages are supposed to use the Layout comp.

Reply:
@mannycolon Thanks for that suggestion - putting it in the Layout component works well.  I tried using _app.js, but trying to reload page from there did not actually reload the page.

Reply:
Glad to help!